### ISSUE TITLE
[Accessibility] ChromeVox navigation issue: accessing visually hidden content

### ISSUE THREAD

**Issue by lpalmaro** [Polymer/docs#1138]
Chrome OS with ChromeVox: When I click on Get Started at the top navigation menu of the home page, I am brought to the Get the code page. When I try to navigate linearly from the top of this page using ChromeVox, there is a problem. I can actually access the same top navigation items (get started, guides and resources, elements, blog} using the screen reader, but they are not visible on the screen (and aren't supposed to be). You will need to apply aria-hidden to the background content, and make sure focus is set properly on the info at hand on this page. This happens across the feature overview and get the code pages, specific to ChromeVox. Since this is only an issue on Chrome OS with ChromeVox, feel free to reach out to dmazzoni@google.com or dtseng@google.com to discuss.

**Comment by arthurevans** [comment]
Can't reproduce. In the current version of the site, the top navigation items are never hidden, so I don't think this issue still applies.
